backend/app/alembic/versions/ec341854b5e6_is_active_field_is_updated_to_bool_only_.py

```
@@ -0,0 +1,99 @@
+
+# revision identifiers, used by Alembic.
+revision = 'ec341854b5e6'
+down_revision = 'aa9cdb105214'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('block', 'is_active',
+               existing_type=sa.BOOLEAN(),
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)